includes/class-wc-stripe-api.php

```diff
@@ -16,6 +16,27 @@ class WC_Stripe_API {
 	const ENDPOINT           = 'https://api.stripe.com/v1/';
 	const STRIPE_API_VERSION = '2024-06-20';
 
+	/**
+	 * The Invalid API Keys cache key.
+	 *
+	 * @var string
+	 */
+	const INVALID_API_KEYS_CACHE_KEY = 'invalid_api_keys_rate_limit';
```

Comment:
I think it would be worth reflecting that the value is a count in the name. It's not obvious to me what data we are caching based on the constant name. Maybe something more like `INVALID_API_RESPONSE_COUNT_CACHE_KEY`?

Reply:
Good point, renamed them to `INVALID_API_KEY_ERROR_COUNT_CACHE_KEY`, `INVALID_API_KEY_ERROR_COUNT_CACHE_TIMEOUT `, and `INVALID_API_KEY_ERROR_COUNT_THRESHOLD ` in [dd3ea04](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4497/commits/dd3ea04e4543deb0f2d6c308e4e6f687c8c69bc2)